uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/OpenTabActionUiTest.scala

```diff
@@ -28,12 +37,19 @@ class OpenTabActionUiTest extends BaseGitRepositoryBackedTest with IntegrationTe
     IntelliJFixture(
       version = IntelliJVersion(version),
       intelliJFactory = IntelliJFactory.Default.withConfig(DriverConfig(vmOptions = Seq("-Xmx1G"))),
-      plugins = Seq(Plugin.Direct(pluginUri)),
+      plugins = Seq(mainPlugin, uiTestActionsPlugin).map(p => Plugin.Direct(p.toURI))
     ).withDisplay().run { ij =>
       ij.probe.openProject(repositoryMainDir)
-      ij.probe.invokeAction("GitMachete.OpenTabAction")
+      // The action we're launching here comes from `uiTestActionsPlugin`,
+      // but it's in turn opening (and peeking into the contents of) the tab that is spawned/controlled by `mainPlugin`.
+      ij.probe.invokeAction("GitMachete.UITest.OpenTabAction")
       ij.probe.awaitIdle()
-      Assert.assertTrue(ij.probe.messages.exists(m => m.level == Info && m.content.endsWith("Opened Git Machete tab: Git Machete")))
+
+      val infoMessages = ij.probe.messages.filter(_.level == Info)
```

Comment:
Missing in Java, `xd => xd.foo()` can be just `_.foo()`; or (`(x, d) => x. foo(d)` can be `_.foo(_)`